pebbles/views/blueprint_templates.py

```diff
@@ -127,6 +129,22 @@ def put(self, template_id):
         db.session.commit()
 
 
+class BlueprintTemplateCopy(restful.Resource):
+    @auth.login_required
+    @requires_admin
+    def put(self, template_id):
+        template = BlueprintTemplate.query.filter_by(id=template_id).first()
+        if not template:
+            abort(404)
+
+        db.session.expunge(template)
+        make_transient(template)
+        template.id = uuid.uuid4().hex
+        template.name = format("%s - %s" % (template.name, 'Copy'))
```

Comment:
I'm a bit worried that people won't rename their copies immediately. Do you think we should enforce e.g. the UUID in the name for copies?